plugins/source/aws/resources/services/wellarchitected/lenses.go

```diff
@@ -0,0 +1,88 @@
+package wellarchitected
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+// lens info consists of types.LensSummary & types.Lens fields
+type lens struct {
+	*types.LensSummary
+	// extra field from types.Lens
+	ShareInvitationId *string
+	// extra field from types.Lens
+	Tags map[string]string
+}
+
+func Lenses() *schema.Table {
+	name := "aws_wellarchitected_lenses"
+	return &schema.Table{
+		Name:        name,
+		Description: `https://docs.aws.amazon.com/wellarchitected/latest/APIReference/API_Lens.html`,
+		Transform: transformers.TransformWithStruct(new(lens),
+			transformers.WithPrimaryKeys("LensAlias"),
+			transformers.WithUnwrapAllEmbeddedStructs(),
+			transformers.WithNameTransformer(client.CreateTrimPrefixTransformer("lens_")),
```

Comment:
I don't think we should be trimming the names...

Reply:
this way we get `arn` column for free as well as `alias` (so it's clear what's happening here)

Reply:
It is only clear if you look and understand the code... Usually we only add new columns with the desired names, not  delete/rename existing ones

Reply:
removed trim in 8c2be0088284f16d121938002b58dc344d100bf9